src/components/vm/vmActions.jsx

```diff
@@ -428,6 +429,20 @@ const VmActions = ({ vm, onAddErrorNotification, isDetailsPage }) => {
         dropdownItems.push(<DropdownSeparator key="separator-migrate" />);
     }
 
+    if (vm.inactiveXML?.hasSpice) {
+        dropdownItems.push(
+            <DropdownItem key={`${id}-replace-spice`}
+                          id={`${id}-replace-spice`}
+                          onClick={() => Dialogs.show(<ReplaceSpiceDialog vmName={vm.name}
+                                                                          vmId={vm.id}
+                                                                          connectionName={vm.connectionName}
+                                                                          vmRunning={vm.state == 'running'} />)}>
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1387-20240122-073835-c9864bc3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)